core-bundle/tests/Contao/StringUtilTest.php

```diff
@@ -308,16 +302,40 @@ public function parseSimpleTokensProvider(): \Generator
             'This is my match',
         ];
 
-        yield 'Test does not support tabs in expressions' => [
-            "This is my {if number\t>\t5}match{endif}",
-            ['number' => 6],
-            'This is my ',
+        yield 'Test if value in array' => [
+            '{if value in ["foobar", "test", "other-value"]}match{else}no-match{endif}',
+            ['value' => 'foobar'],
+            'match',
         ];
 
-        yield 'Test does not support line breaks in expressions' => [
```

Comment:
I've removed the tabs tests because the EL can handle that and I don't see why we wouldn't.